modules/ROOT/pages/bat-example-test-suite.adoc

```diff
@@ -69,7 +69,7 @@ If you write a second test, this one named `second-test.dwl`, you add this line
   file: api-tests/second-test.dwl
 ----
 
-The order in which you list the tests determines the order in which the BAT CLI runs them. In this example, the test `prodInfo.dwl` runs before the test `shoppingCart.dwl`:
+The order in which you list the tests determines the order in which the BAT CLI runs them. In this example, the test `orderFullfillment.dwl` runs before the test `shoppingCart.dwl`:
```

Comment:
Either Fulfillment and Fulfilment are correct words, but not Fullfillment.
If this is not an actual file that can be download and is an illustrative example, I'd suggest rewording to **orderFulfillment.dwl**

Reply:
Yeah, I'll fix it here in this topic, but the developers need to do the same. I'll let them know.